### PR TITLE
Adding core GCMR with bypass flow

### PR DESCRIPTION
**Pull request by licharlot**
Refs #287

### CODE REVIEW

File: microreactors/gcmr/bypass_flow/gold/bypass_out.e
Comment (GiudGiud) [inline microreactors/gcmr/bypass_flow/gold/bypass_out.e:1]: LFS that one or preferably let's reduce the test

Comment (GiudGiud) [reply microreactors/gcmr/bypass_flow/gold/bypass_out.e:1]: Actually the best here would be to add a bunch of postprocessors on the core and use the CSV output as the gold file

Comment (licharlot) [reply microreactors/gcmr/bypass_flow/gold/bypass_out.e:1]: For some unknown reason, the postprocessors don't work well for that type of problem in subchannel.  I've reduced the number of variables in the exodus.

Comment (GiudGiud) [reply microreactors/gcmr/bypass_flow/gold/bypass_out.e:1]: you need to re-write the commits to completely delete the file.

right now the 2mb file is still checked in, as well as this new file

Comment (licharlot) [reply microreactors/gcmr/bypass_flow/gold/bypass_out.e:1]: See https://github.inl.gov/ncrc/subchannel/issues/299 and https://github.inl.gov/ncrc/subchannel/issues/300

Comment (licharlot) [reply microreactors/gcmr/bypass_flow/gold/bypass_out.e:1]: Vasili took a quick look, but didn't find the problem. The values retrned by the PP seem fine when subchannel is the subapp.

Comment (licharlot) [reply microreactors/gcmr/bypass_flow/gold/bypass_out.e:1]: I'll updated the file.

File: .gitattributes [hunk 1]
```diff
@@ -24,3 +24,4 @@ microreactors/mrad/mesh/gold/HPMR_OneSixth_Core_meshgenerator_tri_rotate_bdry.e
 microreactors/mrad/mesh/gold/HPMR_OneSixth_Core_meshgenerator_tri_rotate_bdry_fine.e filter=lfs diff=lfs merge=lfs -text
 microreactors/gcmr/airjacket/mesh/airjacket.re2 filter=lfs diff=lfs merge=lfs -text
 microreactors/gcmr/airjacket/restart_airjacket.fld filter=lfs diff=lfs merge=lfs -text
+microreactors/gcmr/bypass_flow/mesh_bypass_in.e filter=lfs diff=lfs merge=lfs -text
```
Comment (GiudGiud) [inline .gitattributes:27]: this is not needed, we re-generate the mesh in the tests now

**Comment by GiudGiud** [comment]
Times out

How long does it take on your machine? Do you run it with Pronghorn or with subchannel?

**Comment by licharlot** [comment]
> Times out
> 
> How long does it take on your machine? Do you run it with Pronghorn or with subchannel?

I run with Pronghorn, and it takes long on my computer. (5 minutes or so)

File: microreactors/gcmr/bypass_flow/tests [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    executable_pattern = 'pronghorn*'
+    cli_args = '--mesh-only'
+  []
+  [core]
+    type = 'RunApp'
+    input = 'core.i'
+    check_input = True
+    executable_pattern = 'pronghorn*'
+    prereq = 'mesh channel subchannel'
+  []
```
Comment (GiudGiud) [inline microreactors/gcmr/bypass_flow/tests:27]:
```suggestion
    max_time = 500
  []
```

**Comment by GiudGiud** [comment]
This current version of subchannel is broken in devel
https://civet.inl.gov/job/1820363/
@kyriv1980

**Comment by GiudGiud** [comment]
Pronghorn failures are due to the FHR model, unrelated.

**Review by GiudGiud — APPROVED** [review]

**Pull request merged**